tests/system/action/motion/test_create.py

```diff
@@ -290,6 +290,33 @@ def test_create_missing_origin_id(self) -> None:
             "data must not contain {'origin_id'} properties" in response.json["message"]
         )
 
+    def test_create_lead_motion(self) -> None:
```

Comment:
Name is wrong: `create_amendment` since the motion that has a lead motion is an amendment.

Reply:
Renamed.